### ISSUE TITLE
[TF2] Can it be true that if you use "Fabricate Set Headgear" and if that blueprint has multiple possible output cosmetics of different equip regions you only get Hat region items?

### ISSUE THREAD

**Issue by DdariQ** [ValveSoftware/Source-1-Games#4795]
I can't test it until I get another equip region cosmetic since I don't have that many metal, or am I just unlucky? Three times I crafted and I always got Hat region cosmetics, not other region cosmetics, even though those should also be possible outcomes.

I wanted to craft both the Fed-Fightin' Fedora and then the Dillinger's Duffel, but I got the hats both times.
![image](https://user-images.githubusercontent.com/64756077/221123039-82d37c41-12f5-4391-baf0-3fd61b226c3d.png)

Another crafting blueprint with multiple possible output items of different slots also yielded me the hat (hat region cosmetic)
![image](https://user-images.githubusercontent.com/64756077/221123455-e21f6881-bd08-4369-9523-d030195ce0c4.png)

Am I just unlucky or you can't get non-hats/cosmetics that aren't Hat region?

**Comment by JoriKos** [comment]
This seems like a question for the Steam Discussions rather than a GitHub issue.

**Comment by DdariQ** [comment]
True, I was just wondering if there's a chance of it being bugged technically. It's such an obscure issue I'm not sure the public would know the inner workings of this, if it's bugged or not.

**Comment by jh34ghu43gu** [comment]
3 attempts is not significant enough to come to a conclusion about a 50/50 chance, you can easily flip a coin heads 3 times in a row with probably only 2 minutes of attempts at most.
